### ISSUE TITLE
when I use "share" method，it will cause iOS 8.4.1 (12H321) immediately crash (only "share" will happen this，others won't)

### ISSUE THREAD

**Issue by bau720123** [EddyVerbruggen/SocialSharing-PhoneGap-Plugin#422]
hi @EddyVerbruggen 
I am using PGB 5.2.0 + all latest official core plugin + your latest version 5.0.1 from npm (only install your plugin to test it)
https://www.npmjs.com/package/cordova-plugin-x-socialsharing

this is my code
```
window.plugins.socialsharing.share('content', 'title', ['https://www.google.nl/images/srpr/logo4w.png','https://www.google.com.tw/images/nav_logo195.png'], 'http://www.x-services.nl');
```
when I use "share" method，it will cause iOS 8.4.1 (12H321) immediately crash (only "share" will happen this，others won't)

my xcode screenshot
![mac os x 10 10 1-2015-09-21-19-52-24](https://cloud.githubusercontent.com/assets/5110085/9991220/be6d34c2-609a-11e5-96a8-f4c59eae9ed7.png)

**Comment by EddyVerbruggen** [comment]
Hmm, must be caused by #420 - did you also try iOS9?

**Comment by bau720123** [comment]
hi @EddyVerbruggen 
let me upgrade my ipad mini to iOS9 first
feedback later

**Comment by EddyVerbruggen** [comment]
Thanks. In the meanwhile I'm installing an iOS8 simulator ;)

**Comment by EddyVerbruggen** [comment]
iOS sim 8.4 is not a problem as well - sharesheet is popped open and I can choose email. Strange..

**Comment by EddyVerbruggen** [comment]
Oh hang on, it's probably iPad related because #420 did not update the iPad code.

**Comment by EddyVerbruggen** [comment]
Solved in 5.0.2, thx for the report and error (screenshot)

**Comment by bau720123** [comment]
confirm that !!!
thanks @EddyVerbruggen
